DAG-insert-elk-sirene.py

```diff
@@ -100,14 +51,14 @@
 }
 
 with DAG(
-    dag_id=AIRFLOW_DAG_NAME,
+    dag_id=AIRFLOW_DAG_INDEXING_NAME,
     default_args=default_args,
     schedule_interval="0 0 * * 1,3,5",
```

Comment:
@HAEKADI , you might want to automatically trigger this DAG when the DAG-preprocess-data succeeds.
It seems to be doable : https://docs.astronomer.io/learn/cross-dag-dependencies#dataset-dependencies

Reply:
I have another PR to reorganise how DAGs are ran, because the main indexation DAG does not only depend on this one.

Reply:
https://github.com/etalab/annuaire-entreprises-search-infra/pull/230